components/service/experiments/README.md

```diff
@@ -12,137 +12,44 @@ Use Gradle to download the library from [maven.mozilla.org](https://maven.mozill
 implementation "org.mozilla.components:service-experiments:{latest-version}"
 ```
 
-### Creating an Experiments instance
-In order to use the library, first you have to create a new `Experiments` instance. You do this once per app launch 
-(typically in your `Application` class `onCreate` method). You simply have to instantiate the `Experiments` class and
-provide the `ExperimentStorage` and `ExperimentSource` implementations, like this:
+### Initializing the Experiments library
+In order to use the library, first you have to initialize it by calling `Experiments.initialize()`. You do this once per app launch 
+(typically in your `Application` class `onCreate` method). You simply have to call `Experiments.initialize()` and
+provide the `applicationContext` (and optionally a `Configuration` object), like this:
 
 ```Kotlin
 class SampleApp : Application() {
-    lateinit var experiments: Experiments
-
     override fun onCreate() {
-        experiments = Experiments(
-            experimentSource,
-            experimentStorage
+        // Glean needs to be initialized first.
+        Glean.initialize(/* ... */)
+        Experiments.initialize(
+            applicationContext,
+            configuration // This is optional, e.g. for overriding the fetch client.
         )
     }
 }
 ```
 
-#### Using Kinto as experiment source
-Experiments includes a default source implementation for a Kinto backend, which you can use like this:
+Note that this library depends on the Glean library, which has to be updated first. See the [Glean README](../glean/README.md) for more details.
```

Comment:
What do you mean by glean needs to be "updated first"?

Reply:
That should be "initialized".